contracts/src/token/erc20/extensions/permit.rs

```diff
@@ -60,18 +60,6 @@ pub enum Error {
     ECDSA(ecdsa::Error),
 }
 
-impl MethodError for erc20::Error {
```

Comment:
`MethodError` implementation is moved to the corresponding contract

Reply:
This means that this import can be removed from this file, right?